platforms/jvm/language-jvm/src/main/java/org/gradle/api/tasks/compile/AbstractCompile.java

```diff
@@ -78,8 +78,8 @@ public void setClasspath(FileCollection configuration) {
     @OutputDirectory
     @ReplacesEagerProperty(
         originalAccessors = {
-            @UpgradedAccessor(value = GETTER, methodName = "getDestinationDir"),
-            @UpgradedAccessor(value = SETTER, methodName = "setDestinationDir")
+            @UpgradedAccessor(value = GETTER, name = "getDestinationDir"),
+            @UpgradedAccessor(value = SETTER, name = "setDestinationDir")
```

Comment:
❓ Should we also rename `@UpgradedAccessor` to `@ReplacedAccessor` and `@UpgradedDeprecation` to... something else?

Reply:
Hmm, not sure what is better. I kinda got used to `upgraded`, but I guess also `ReplacedAccessor` could work. It is a bit more generic, unless we use `@EagerAccessor`.

What about `@UpgradedDeprecation`?